File: MainApp/Models/Airport.cs
using System.Collections.Generic;

namespace MainApp.Models
{
    public class Airport
    {
        public int AirportId {get;set;}
        public int AirportRef {get;set;}
        public string AirportName {get;set;}
        public double Latitude {get;set;}
        public double Longitude {get;set;}
        public List<Runway> Runways {get;set;}
    }
}

File: MainApp/ClientApp/src/app/dtos/Airport.ts
import { WeatherForecast } from "./WeatherForecast";
import { Runway } from "./Runway";

export interface Airport {
    airportId: number;
    airportName: string;
    latitude: string;
    longitude: string;
    runways: Runway[];
    weatherForecast: WeatherForecast[];
  }